src/main/java/dev/j3fftw/litexpansion/armor/SolarHelmets.java

```diff
@@ -0,0 +1,77 @@
+package dev.j3fftw.litexpansion.armor;
+
+import dev.j3fftw.litexpansion.Items;
+import io.github.thebusybiscuit.slimefun4.implementation.SlimefunItems;
+import io.github.thebusybiscuit.slimefun4.implementation.items.electric.gadgets.SolarHelmet;
+import javax.annotation.Nonnull;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import me.mrCookieSlime.Slimefun.Lists.RecipeType;
+import me.mrCookieSlime.Slimefun.api.SlimefunItemStack;
+import org.bukkit.enchantments.Enchantment;
+import org.bukkit.inventory.ItemStack;
+
+public class SolarHelmets extends SolarHelmet {
```

Comment:
Conventions :NotLikeThis: No plural

```suggestion
public class AdvancedSolarHelmet extends SolarHelmet {
```

Reply:
is this an rule or something havent heard that before